llvm/test/CodeGen/AIE/aie2/GlobalISel/inst-select-no-combine-vldb_unpack.mir

```diff
@@ -10,24 +10,23 @@
 
 # This tests that we don't combine if one of the load's defs is used before the VUNPACK instruction
```

Comment:
That comment is outdated. Now we do combine 💪

Reply:
Moved to the other file (with positive tests)